src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/MigrateVisitRequestDto.kt

```diff
@@ -26,6 +27,9 @@ data class MigrateVisitRequestDto(
   @Schema(description = "Visit Status", example = "RESERVED", required = true)
   @field:NotNull
   val visitStatus: VisitStatus,
+  @Schema(description = "Outcome Status", example = "RESERVED", required = true)
+  @field:NotNull
```

Comment:
may be this should be taken out?

Reply:
What is the thought behind adding it?  We are migrating visits which have not completed yet

Reply:
I should take it out right?

Reply:
Required -> false, remove not null annotation, add schema 'defaultValue =' to match impl